yarn-project/p2p/src/services/peer-manager/peer_manager.ts

```diff
@@ -163,13 +180,13 @@ export class PeerManager implements PeerManagerInterface {
    */
   private handleConnectedPeerEvent(e: CustomEvent<PeerId>) {
```

Comment:
Not from this PR, but I was wondering if we are sending twice the status handshakes? If this event fires for both peers when a connection is established between them, then both of them initiate the status handshake, and both of them respond with their own status.